app/settings/__init__.py

```diff
@@ -17,6 +17,8 @@ def get_settings():
         set_settings(secret='super secret key', app_name='Open Event')
     else:
         current_app.config['custom_settings'] = make_dict(s)
+        if not current_app.config['custom_settings']['secret']:
```

Comment:
No, not this way. You should use `get()` method else it will throw KeyError it is not defined in this case.

`if not current_app.config['custom_settings'].get('secret')`